connector-runtime/spring-boot-starter-camunda-connectors/src/main/java/io/camunda/connector/runtime/OutboundConnectorsAutoConfiguration.java

```diff
@@ -81,8 +95,41 @@ public SecretProviderAggregator springSecretProviderAggregator(
       name = "camunda.connector.secretprovider.environment.enabled",
       havingValue = "true",
       matchIfMissing = true)
-  public SpringEnvironmentSecretProvider defaultSecretProvider(Environment environment) {
-    return new SpringEnvironmentSecretProvider(environment, environmentSecretProviderPrefix);
+  public EnvironmentSecretProvider defaultSecretProvider(Environment environment) {
+    return new EnvironmentSecretProvider(environment, environmentSecretProviderPrefix);
+  }
+
+  @Bean
+  @ConditionalOnProperty(
+      name = "camunda.connector.secretprovider.console.enabled",
+      havingValue = "true")
+  public ConsoleSecretProvider consoleSecretProvider(
+      ConsoleSecretApiClient consoleSecretApiClient) {
+    return new ConsoleSecretProvider(consoleSecretApiClient, Duration.ofSeconds(20));
+  }
+
+  @Bean
+  @ConditionalOnProperty(
+      name = "camunda.connector.secretprovider.console.enabled",
```

Comment:
I know it works, but should we use **secret-provider** instead of **secretprovider**, could be useful some day when searching the code for these properties